src/main/java/com/home/MyWizardBot/MyWizardTelegramBot.java

```diff
@@ -0,0 +1,49 @@
+package com.home.MyWizardBot;
+
+import lombok.Setter;
+import org.telegram.telegrambots.bots.DefaultBotOptions;
+import org.telegram.telegrambots.bots.TelegramWebhookBot;
+import org.telegram.telegrambots.meta.api.methods.BotApiMethod;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+@Setter
+public class MyWizardTelegramBot extends TelegramWebhookBot {
+
+    private String webHookPath;
+    private String botUserName;
+    private String botToken;
+
+    @Override
+    public BotApiMethod<?> onWebhookUpdateReceived(Update update) {
+        // We check if the update has a message and the message has text
+        if (update.getMessage() != null && update.getMessage().hasText()) {
+            // Set variables
+            long chatID = update.getMessage().getChatId();
```

Comment:
Try to stick to camelcase when naming variables. `chatId` better than `chatID`.

Reply:
Fixed ChatID variable name to Chat_Id